app/data_models/progress_store.py

```diff
@@ -138,6 +138,15 @@ def get_section_status(
 
         return CompletionStatus.NOT_STARTED
 
+    def get_block_status(
+        self, block_id: str, section_id: str, list_item_id: str | None
```

Comment:
I suggest enforcing keyword args so the order can't be mistaken.

Reply:
Added keywords enforcement